docs/slack.rst

```diff
@@ -11,6 +11,11 @@ Below we have a bit more information about the useful Slack channels that we hav
 This should start you off on the right foot,
 and get you connected with folks who have similar interests.
 
+Our network uses the Slack "Free" plan. Under this plan, Slack limits access to the latest
+10k messages. For us, this is a feature, as many people seem to feel a freedom in knowing
+that their words will disappear shortly. And many have said that if the conversations were
+archived/published, they’d quit participating.
+
```

Comment:
I might rephrase this a bit, because cost is also a factor, and I think folks are pretty blissfully unaware of just how expensive paid Slack can be. (also please remove quotes around "free" ;) ) Suggestion:

We don't pay for our Slack, which means that the community has access to only the most recent 10,000 messages. Not only is paid Slack prohibitively expensive for a community like ours, but many people seem to feel a freedom in knowing that their words will disappear shortly. Folks have also said that if the conversations were archived/published, they’d quit participating.

(we could expand, but that would probably get into too many more words, and this is mostly for reference anyway, right? Most people won't read it, and we'll undoubtedly continue to get questions about it, which we'll continue to field. In the same spirit as the conversation that inspired this addition :D )

Reply:
wfm, will update as such.